lib/jurnal_api/client/sales_invoice_payments.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module JurnalApi
+  class Client
+    # Defines methods related to ReceivePayments from SalesInvoice
+    module SalesInvoicePayments
+
+      def sales_invoice_payments(id, params = {})
+        response = get("sales_invoices/#{id}/receive_payments", params)
```

Comment:
I think we can make it inline, no needs to assign it to the `response` variable.